terraform/environment/region/event_bus.tf

```diff
@@ -0,0 +1,8 @@
+module "event_bus" {
```

Comment:
At the moment, it only provisions an event bus (and forwarding rule) and not events. There's another module that handles receiving events and it could get confusing the other way

Reply:
Makes sense 👍